src/menu/menu.c

```diff
@@ -19,12 +20,15 @@
 #include "common/nodename.h"
 #include "common/scaled_font_buffer.h"
 #include "common/scene-helpers.h"
+#include "common/spawn.h"
 #include "common/string-helpers.h"
 #include "labwc.h"
 #include "menu/menu.h"
 #include "node.h"
 #include "theme.h"
 
+#define PIPEMENU_TIMEOUT_IN_MS 4000
```

Comment:
Should we make this user controllable, e.g. via some kind of config (or theme) option?

Reply:
Suggest start hard-coded for now and add if there is a request for it.